Assets/MixedRealityToolkit/_Core/Inspectors/MixedRealityConfigurationProfileInspector.cs

```diff
@@ -149,22 +165,20 @@ public override void OnInspectorGUI()
             {
                 EditorGUILayout.PropertyField(inputSystemType);
                 RenderProfile(inputActionsProfile);
+                //RenderProfile(pointerProfile);
```

Comment:
why is this commented out?

Reply:
Cause I didn't add it yet.  (Must have been left over when I ported it over) I'll remove it.